tests/utils/basic.inc

```diff
@@ -1,38 +1,8 @@
 <?php
-require __DIR__ . "/" . "tools.php";
 
-if (($FILENAME = getenv("PHONGO_SERVERS")) === false) {
-    $FILENAME = sys_get_temp_dir() . "/PHONGO-SERVERS.json";
-}
+require_once __DIR__ . "/" . "tools.php";
 
-$json = file_get_contents($FILENAME);
-$config = json_decode($json, true);
-if (!$config) {
-    exit("skip Couldn't json_decode(file_get_contents($FILENAME));");
-}
-
-$servers = array(
-    "STANDALONE"      => "",
-    "STANDALONE_30"   => "",
-    "STANDALONE_SSL"  => "",
-    "STANDALONE_AUTH" => "",
-    "STANDALONE_X509" => "",
-    "STANDALONE_PLAIN" => "",
-    "REPLICASET"       => "",
-    "REPLICASET_30"    => "",
-    "REPLICASET_DNS"   => "",
-);
-$servers = array_merge($servers, $config);
-def($servers);
-
-$consts = array(
-    "DATABASE_NAME"                 => "phongo",
-    "COLLECTION_NAME"               => makeCollectionNameFromFilename($_SERVER["SCRIPT_FILENAME"]),
-);
-def($consts);
-
-// These use values from constants defined above
-$consts = array(
-    "NS"             => DATABASE_NAME . "." . COLLECTION_NAME,
-);
-def($consts);
+define('URI', getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
```

Comment:
This is going to require changes to `make test`, which I intend to document in `CONTRIBUTING.md`. The C driver takes a similar approach: https://github.com/mongodb/mongo-c-driver/blob/master/CONTRIBUTING.md#testing